src/sgame/botlib/bot_load.cpp

```diff
@@ -68,6 +71,27 @@ void BotInit()
 #endif
 }
 
+void BotPrintOffMeshConnections( OffMeshConnections &con )
```

Comment:
Remove this. Log verbosity must be controlled by cvars, not build type. But it's too noisy in any case.

Reply:
I really don't want to ever have to debug this code without that debug log.

Reply:
What would be the log be like if the entire contents of every file the game opens were printed to the console? It's not reasonable. This has a very low chance of being useful to anyone for how spammy it is.

Reply:
At first it was just a code I needed in order to do that PR and verify the code works, including when implementing suggested changes like the new way to parse lines. Right now this debug log is only printed if debug log is enabled and this is a debug build, this can be made even more disabled than that, but really, I don't want to edit this code without any way to check for the correct parsing.

Every suggest or request in favor of a rewrite of a part of this parser is just asking for this debug log to exist, even if hidden behind a `#if 0`.

Reply:
When you're debugging and want to see it you can hack in a call to `BotSaveOffMeshConnections`, which prints exactly the same thing, and look at the file it generates.

Or just rewrite it which would take all of 5 minutes.

Reply:
> I added a commit to delete the extra debug print, that commit is easy to revert for re-enabling debugging. -- https://github.com/Unvanquished/Unvanquished/pull/2329#issuecomment-1369301032